index.js

```diff
@@ -421,6 +422,10 @@ function slowToString (encoding, start, end) {
   }
 }
 
+// Even though this property is private, it shouldn't be removed because it is
+// used by `is-buffer` to detect buffer instances in Safari 5-7.
+Buffer.prototype._isBuffer = true
```

Comment:
This was only being set in the fallback case originally, why is it now *always* set?